packages/markdown/commands.lua

```diff
@@ -7,6 +7,7 @@
 --
 local utils = require("packages.markdown.utils")
 local base = require("packages.base")
+local html = require("packages.markdown.html")
```

Comment:
In `packages/markdown/init.lua` you are loading it via the class, so there wouldn't be any reason to load it too via `require()`, is there? (and the commands would be registered via the class if you rename your `registerHTMLcommands` to `registerCommands`, simply.